src/python/arcor2_arserver/decorators.py

```diff
@@ -48,3 +49,38 @@ async def async_wrapper(*args, **kwargs) -> Any:
         return await coro(*args, **kwargs)
 
     return cast(F, async_wrapper)
+
+
+def retry(exc=Exception, tries: int = 1, delay: float = 0):
```

Comment:
Tady spíš nějaký `LockingException` (potomek `Arcor2Exception` a rodič `CannotUnlock` a `CannotLock`) než `Exception`?

Reply:
Tady bych to možná nechal obecně a u dekorované funkce bych to změnil? Pak to někdo použije znovu a nemusí to fungovat intuitivně

Reply:
Tak obecně by právě bylo to `LockingException` a na konkrétních místech by si to vývojář změnil na `CannotUnlock`, nebo `CannotLock`. `Exception` je nebezpečná výchozí hodnota. Pokud by to náhodou někdo nezměnil, můžou se dít věci... Odchytávat `Exception` by se mělo jen ve velmi velmi speciálních případech.